src/components/accession2/viabilityTesting/ViabilityTestingPanel.tsx

```diff
@@ -15,15 +15,16 @@ type ViabilityTestingPanelProps = {
   user: User;
   setNewViabilityTestOpened: React.Dispatch<React.SetStateAction<boolean>>;
   setSelectedTest: React.Dispatch<React.SetStateAction<ViabilityTest | undefined>>;
+  setViewViabilityTestModalOpened: React.Dispatch<React.SetStateAction<boolean>>;
 };
 
 export default function ViabilityTestingPanel(props: ViabilityTestingPanelProps): JSX.Element {
-  const { accession, setNewViabilityTestOpened, setSelectedTest } = props;
+  const { accession, setNewViabilityTestOpened, setSelectedTest, setViewViabilityTestModalOpened } = props;
```

Comment:
i have a similar hook in my database PR, after I merge can you rebase and hook it up?

Reply:
Yes! I will rebase all prs after you merge

Reply:
I did all the rebase, but what is the hook you are referring to?

Reply:
oh i meant the callback when user clicks on test in the database - you have already connected that, we are good 👍